modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/RelatedObjectEntryResourceTest.java

```diff
@@ -141,6 +141,78 @@ public void tearDown() throws Exception {
 			_objectRelationship);
 	}
 
+	@Test
+	public void testDeleteNotFoundSystemObjectRelatedObject() throws Exception {
+		Long irrelevantPrimaryKey = RandomTestUtil.randomLong();
+
+		JSONObject jsonObject = _invoke(
+			StringBundler.concat(
+				_userSystemObjectDefinitionMetadata.getRESTContextPath(),
+				StringPool.SLASH, _user.getUserId(), StringPool.SLASH,
+				_objectRelationship.getName()),
+			Http.Method.GET);
+
+		JSONArray itemsJSONArray = jsonObject.getJSONArray("items");
+
+		Assert.assertEquals(1, itemsJSONArray.length());
+
+		_invoke(
+			StringBundler.concat(
+				_userSystemObjectDefinitionMetadata.getRESTContextPath(),
+				StringPool.SLASH, _user.getUserId(), StringPool.SLASH,
+				_objectRelationship.getName(), StringPool.SLASH,
+				irrelevantPrimaryKey),
+			Http.Method.DELETE);
+
+		jsonObject = _invoke(
+			StringBundler.concat(
+				_userSystemObjectDefinitionMetadata.getRESTContextPath(),
+				StringPool.SLASH, _user.getUserId(), StringPool.SLASH,
+				_objectRelationship.getName()),
+			Http.Method.GET);
+
+		itemsJSONArray = jsonObject.getJSONArray("items");
+
+		Assert.assertEquals(1, itemsJSONArray.length());
+	}
+
+	@Test
+	public void testDeleteSystemObjectRelatedObject() throws Exception {
+		String objectFieldValue = RandomTestUtil.randomString();
+
+		ObjectEntry objectEntry = _addObjectEntry(objectFieldValue);
```

Comment:
in _objectEntry you already has an object entry created and that objectEntry has been already related to the user, so this code is not necessary, you could use those elements